sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -285,9 +333,12 @@ async def create_container(
         data = await self.client_connection.CreateContainer(
             database_link=self.database_link, collection=definition, options=request_options, **kwargs
         )
-        return ContainerProxy(self.client_connection, self.database_link, data["id"], properties=data)
+        if not return_headers:
+            return ContainerProxy(self.client_connection, self.database_link, data["id"], properties=data)
```

Comment:
```suggestion
            return ContainerProxy(self.client_connection, self.database_link, data["id"], properties=data, header=data.get_response_headers())
```

Why don't we add header here, so `ContainerProxy` can be used to get header?